packages/test/test-end-to-end-tests/src/test/localTestSignals.spec.ts

```diff
@@ -109,6 +111,10 @@ describeCompat("TestSignals", "FullCompat", (getTestObjectProvider) => {
 		});
 	});
 
+	it.skip("Lodash", () => {
+		assert.equal(_.sum([0, 0]), 0);
```

Comment:
this doesn't make any sense to me. we don't bundle our tests. why does this impact bundle size at all

Reply:
from the listed bundle analysis the total only goes up by ~150 bytes, which i think is expected fluctuation.

Reply:
this was just to see if adding the label and warning message worked